app/assets/javascripts/components/common/article_viewer.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import OnClickOutside from 'react-onclickoutside';
 import { trunc } from '../../utils/strings';
+import SUPPORTED_LANG from '../../constants/mediawiki_lang';
```

Comment:
Since this is only used in ArticleViewer, we may as well just do this in the same file. Also, whocolor is different from mediawiki (which is the software that runs Wikipedia and other wikis).

Reply:
I have removed this extra const file.